data/hai/hai ships.txt

```diff
@@ -1075,6 +1075,77 @@ ship "Shield Beetle" "Shield Beetle (Tracker)"
 
 
 
+# Flagship of the Unfettered Hai, filled with the spoils of wars against Korath and Wanderers:
+ship "Emperor Beetle"
+	swizzle 6
+	sprite "ship/hai emperor beetle"
+	thumbnail "thumbnail/hai emperor beetle"
+	attributes
+		category "Heavy Warship"
+		"cost" 25100000
+		"shields" 37200
+		"hull" 11780
+		"required crew" 62
+		"bunks" 70
+		"mass" 725
+		"drag" 9.5
+		"heat dissipation" .4
+		"fuel capacity" 500
+		"outfit space" 872
+		"weapon capacity" 333
+		"engine capacity" 150
+		weapon
+			"blast radius" 500
+			"shield damage" 5000
+			"hull damage" 2500
+			"hit force" 3500
+	outfits
+		"Large Heat Shunt" 2
+		"Jump Drive"
+		"Systems Core (Large)"
+		"Systems Core (Medium)"
+		"Quantum Keystone"
+		"Geode Reactor"
+		"White Sun Reactor"
+		`"Bufaer" Atomic Steering`
```

Comment:
Using Sunbeams (or any beam weapon) effectively against a Korath Banisher or fast pirate ship requires excellent turning, acceleration, and top speed. Increasing the drag without adding engines will make the sunbeams far less useful.

Reply:
We also have less sunbeams and the drag diff isnt significant
This is not intended to fight korath or wanderer. The Uhai outfit it differently each time they go to war

Reply:
In my tests, your outfits have plenty of thrust and steering, so the drag and mass are fine.